fixtures.go

```diff
@@ -157,6 +157,12 @@ var fixtures = Fixtures{{
 }, {
 	Tags:       []string{"merge-base"},
 	DotGitHash: plumbing.NewHash("26baa505b9f6fb2024b9999c140b75514718c988"),
+}, {
+	Tags:         []string{"commitgraph"},
```

Comment:
Can you rename  the tag to `commit-graph`?

Reply:
Done